lib/Doctrine/ORM/Query.php

```diff
@@ -322,6 +322,12 @@ protected function _doExecute()
 
         list($sqlParams, $types) = $this->processParameterMappings($paramMappings);
 
+        if ($this->_queryCacheProfile && $this->getExpireResultCache()) {
+            $this->_queryCacheProfile->getResultCacheDriver()->delete(
+                $this->_queryCacheProfile->generateCacheKeys($executor->getSqlStatements(), $sqlParams, $types)[0]
```

Comment:
I thought that but although the docblock says it returns an `array` it's actually a string 😱